gulpfile.js

```diff
@@ -192,11 +191,20 @@ gulp.task("build:copy", () => {
 
     // Get the config.js file for the task to know what scanner to use
     const configJs = path.join(BUILD_TS_DIR, "common", commonPath, "config.js");
+
+    console.log("configJs", configJs);
     // eslint-disable-next-line import/no-dynamic-require
     const { scanner } = require(configJs);
 
     // Copy Scanner CLI
-    if (taskNeedsCliScanner(taskName)) {
+    const versionRequiresBundledScanners = ["sonarcloud@v1", "sonarqube@v4", "sonarqube@v5"];
+    const currentExtensionVersion = `${extension}@${version}`;
+    console.log(currentExtensionVersion + ":" + taskName);
+    if (
+      taskNeedsCliScanner(taskName) &&
+      versionRequiresBundledScanners.includes(currentExtensionVersion)
```

Comment:
Then we can access the boolean `scanner.embedScanners` to know whether we should bundle the scanners (if not, we can skip both these ifs) - wdyt?